packages/shared/src/styles/components/buttons.css

```
@@ -129,15 +156,7 @@
     --button-box-shadow: var(--button-pressed-box-shadow);
   }
 
-  &:not(.readOnly) {
-    &:hover, &.hover, &:focus:focus-visible {
-      --button-background: var(--button-hover-background);
-      --button-border-color: var(--button-hover-border-color);
-      --button-color: var(--button-hover-color);
-      --button-box-shadow: var(--button-hover-box-shadow);
-    }
-  }
-
+ 
```

Just reverting the placement here.

```suggestion
 &:not(.readOnly) {
    &:hover, &.hover, &:focus:focus-visible {
      --button-background: var(--button-hover-background);
      --button-border-color: var(--button-hover-border-color);
      --button-color: var(--button-hover-color);
      --button-box-shadow: var(--button-hover-box-shadow);
    }
  }
  
```